client/signup/steps/domains/index.jsx

```diff
@@ -276,25 +290,33 @@ class DomainsStep extends React.Component {
 	}
 
 	domainForm = () => {
-		const initialState = this.props.step ? this.props.step.domainForm : this.state.domainForm;
+		let initialState = {};
+		if ( this.state ) {
+			initialState = this.state.domainForm;
+		}
+		if ( this.props.step ) {
+			initialState = this.props.step.domainForm;
+		}
 
 		return (
 			<RegisterDomainStep
```

Comment:
We recently [refactored this component](https://github.com/Automattic/wp-calypso/pull/27035) to receive a `vendor` property. 

Do we intend to apply `vendor={ abtest( 'domainSuggestionKrakenV325' ) }` here? Doing so will apply the A/B test for both signup and `/domains` if I'm reading this code correctly.

Reply:
Given that we recently closed out all A/B tests with #27165, this specific point is moot. So, to reframe my question: do we have any interest in running vendor A/B tests in `/domains`? If so, what plans do we have for this?

Reply:
No idea :D But it's probably a discussion best suited for a P2 :)

Reply:
If need be, we can test them separately, I guess.